CONTRIBUTING.md

```diff
@@ -93,6 +93,8 @@ If you've found something in the content or the website that should be updated,
 ### :hammer_and_wrench: Pull requests
 A [pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests) is a way to suggest changes in our repository.
 
+`note`: `rest` content is autogenerated and should not be modified by mortals.
```

Comment:
```suggestion
We cannot accept contributions to the [REST API reference documentation](https://docs.github.com/en/rest/reference). If you spot an inaccuracy in the REST API reference documentation, open an issue in the [github/rest-api-description](https://github.com/github/rest-api-description/issues/new?template=schema-inaccuracy.md) repository.
```
👋 @yashhd Congratulations on your first open source contribution and thank you so much for opening a PR 🎉 🙌  

One of the writers added a line in this other section https://github.com/github/docs/blob/main/CONTRIBUTING.md#dont-see-your-issue-open-one that I think would work perfectly here! I added it as a suggestion above ☝️ 

Instead of adding the line here, let's put it down after the next sentence on line 98.  If you ping me in once that's done, we'll get this merged 🎉

Reply:
@janiceilene I have committed the changes as you mentioned above. I think it is good to merge now. Thank you so much for this opportunity! Got a start in this open-source community.